B/Binutils/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder
 
 name = "Binutils"
-version = v"2.41"
+version = v"2.41.1"
```

Comment:
Don't need to change the version number for this

Reply:
Done.